src/components/Button.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+function Button({ onClick }) {
+  return [1, 2, 3, 4, 5].map((i) => (
+    <button type="button" key={i} onClick={() => onClick(i)}>
```

Comment:
```suggestion
    <button
         type="button"
         key={i}
         onClick={() => onClick(i)}
     >
```

저는 `props` 가 많아지면 이렇게 줄바꿈해서 formatting 해주는 편인데 어떤게 더 가독성이 좋게 느껴지시나요?

Reply:
ㅇㅎ 줄바꿈을 하는게 한눈에 잘 들어옵니다.